src/env.cc

```diff
@@ -933,6 +948,52 @@ char* Environment::Reallocate(char* data, size_t old_size, size_t size) {
   return new_data;
 }
 
+void AsyncRequest::Install(Environment* env, void* data, uv_async_cb target) {
+  Mutex::ScopedLock lock(mutex_);
+  env_ = env;
+  async_ = new uv_async_t;
+  if (data != nullptr) async_->data = data;
```

Comment:
I’d maybe remove the conditional here, it probably still makes sense to set `data` even if it is `nullptr`?

Reply:
@addaleax -done.